launch_testing/README.md

```diff
@@ -201,6 +201,18 @@ add_launch_test(
 
 ## Examples
 
+### `hello_world_launch_test.py`
+
+Usage:
+
+```sh
+launch_test test/launch_testing/examples/hello_world_launch_test.py
+```
+
+This test is meant to be a simple example on how to use the ``launch_testing``. It launches a process to print "hello_world" in the ``stdout`` stream,
+and uses ``proc_output.assertWaitFor()`` to detect the text. Finally it checks if the original process exited normally.
+The ``@launch_testing.markers.keep_alive`` decorator makes sure that the launch process stays alive for the tests.
```

Comment:
Here are some wording suggestions. Also, use one sentence per line to help keep future diffs small #nitpick 

```suggestion
This test is a simple example on how to use ``launch_testing``.
It launches a process and asserts that it prints "hello_world" to ``stdout`` using ``proc_output.assertWaitFor()``.
Finally, it checks if the process exits normally (zero exit code).

The ``@launch_testing.markers.keep_alive`` decorator ensures that the launch process stays alive long enough for the tests to run.
```